### PR TITLE
[LPT] Tile transformation

### PR DESCRIPTION
**Pull request by vzinovie**
### Tickets:
 - 67521

### CODE REVIEW

**Review by eshoguli — COMMENTED** [review]
still in WIP

File: src/common/low_precision_transformations/src/fake_quantize_dequantization.cpp [hunk 1]
```diff
@@ -179,6 +179,19 @@ std::shared_ptr<Node> FakeQuantizeDequantization::copyWithNewInput(const std::sh
     return lastNode;
 }
 
+void FakeQuantizeDequantization::scalarizeConstants() {
+    auto replaceDeqConstToScalar = [](std::shared_ptr<opset1::Constant> constant) {
+        if (constant && ov::shape_size(constant->get_shape()) != 1) {
+            auto scalarConstant = ov::as_type_ptr<opset1::Constant>(NetworkHelper::toScalarIfPossible(constant));
```
[the remark this answers is not in the document]
Comment (vzinovie) [reply src/common/low_precision_transformations/src/fake_quantize_dequantization.cpp:185]: > can we don't replace constant if it is not changed?

@eshoguli Actually, the check on line 184 is exactly what you mean.

[the remark this answers is not in the document]
Comment (vzinovie) [reply src/common/low_precision_transformations/src/fake_quantize_dequantization.cpp:185]: Thanks, done!

File: src/common/low_precision_transformations/include/low_precision/tile.hpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2022 Intel Corporation
```
[the remark this answers is not in the document]
Comment (vzinovie) [reply src/common/low_precision_transformations/include/low_precision/tile.hpp]: Thanks, done!

**Review by eshoguli — COMMENTED** [review]
Why multichannels quantization are not supported?

**Review by eshoguli — APPROVED** [review]
LGTM

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**